index.js

```diff
@@ -27,4 +28,19 @@ app.get('/', (req, res) => {
   );
 });
 
-app.listen(port, () => console.log(`Express App Listening on port ${port}!`));
+app.post('/invite', async (req, res) => {
+  const RECORD_ID = req.body.pledgeInviteID;
+  const confirmSend = await sendInviteEmail(RECORD_ID);
+
+  if (confirmSend === '') {
+    res.send({
+      status: `An error occured when sending an invitation.`
```

Comment:
by default, I think `res.send` sends a response with a `200` code, which is generally consider okay. But since there was an error, we should send a something like a `400` code. In Express you can do this by doing something like: 
`res.status(400).send({status: "Error ocurred"})`

Reply:
Honestly, you don't need to change it, just a note for your info honestly lol. We're not using status codes properly anywhere else either so maybe when there's an error handling PR we'll be more meticulous. More on status codes here: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors